.github/ISSUE_TEMPLATE/api-bug-report.yml

```diff
@@ -47,3 +47,10 @@ body:
       description: What is the browser/library/client you are using? What operating system and version?
     validations:
       required: true
+  - type: textarea
+    id: additionalContext
+    attributes:
+      label: Additional Context
+      description: Additional Context/Informations
+    validations:
+      required: true
```

Comment:
You can just use the description, also why is this required?

Reply:
> required

Yeah forgot it, it should not be required but in description you just have to describe the bug so that's why I think they should add a Additional Context element

Reply:
any important relevant information should be in the description. also, `Informations` is incorrect